### PR TITLE
komorebi@0.1.29: Update license

### PR DESCRIPTION
**Pull request by LGUG2Z**
Ensures that the license accurately reflects the [new license used for v0.1.29](https://github.com/LGUG2Z/komorebi/blob/v0.1.29/LICENSE.md).

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by thesobercoder** [comment]
Please merge this.

File: bucket/komorebi.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "version": "0.1.29",
     "description": "A tiling window manager for Windows",
     "homepage": "https://github.com/LGUG2Z/komorebi",
-    "license": "PolyForm Strict",
+    "license": "Komorebi",
```
Comment (aliesbelik) [inline bucket/komorebi.json:5]:
```suggestion
    "license": {
        "identifier": "Komorebi",
        "url": "https://github.com/LGUG2Z/komorebi/blob/master/LICENSE.md"
    },
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
For non-SPDX licenses, include a link to the license.

**Review by aliesbelik — APPROVED** [review]

**Comment by nahid6970** [comment]
why the hell its still not online ?? its already online in winget months ago. is the scoop maintainers having some personal issues? i thought scoop package managers is millions time better than winget. but oh i was so wrong. a version that is already updated to winget month ago and here we are.

**Pull request merged**